bio_hansel/const.py

```diff
@@ -48,15 +48,21 @@
 n_tiles_matching_subtype
 n_tiles_matching_subtype_expected
 file_path
+coverage
```

Comment:
We may need to make this a little more specific. How about `avg_tile_coverage`? We're using it as a proxy for the actual sequencing depth of coverage.

Reply:
Agreed, I'll change it to that.